config/kubernetes.spc

```diff
@@ -12,6 +12,13 @@ connection "kubernetes" {
   # Specify a context other than the current one.
   # config_context = "minikube"
 
+  # Specify the custom resource definitions for which the dynamic tables will be created.
+  # The custom_resource_tables list may include wildcards (e.g. *, ip*, storagestates.migration.???.io), singular names or the full name.
+  # By default plugin will create the dynamic tables for all the available custom resource definitions.
+  # The plugin will not create dynamic tables if custom_resource_tables is empty or not set.
+  # custom_resource_tables = ["certificate","ip*","storagestates.migration.k8s.io"]
+  custom_resource_tables = ["*"]
```

Comment:
```suggestion
  # List of custom resource definitions that will be created as dynamic tables
  # Wildcard based searches are supported
  # No dynamic tables will be created if this arg is empty or not set
  # Defaults to all custom resource definitions
  custom_resource_tables = ["*"]
```

Instead of the last commented line showing an example (removed in the suggestion above), can you please add some examples showing how to match on the full and short name like in https://github.com/turbot/steampipe-plugin-csv/blob/main/config/csv.spc#L10-L17?

Also, in our language, should we use `custom resource definitions` or `custom resources`, since the arg is called `custom_resource_tables`, not `custom_resource_definition_tables`?

Reply:
For the long and short names I mentioned above, how does Kube refer to these?

Reply:
The long name is the custom resource definition name like - `certificates.cert-manager.io`. The format is
spec.names.plural+"."+spec.group.
The short name is the custom resource definition singular name like - `certificates.cert-manager.io`. The property is
spec.names.singular.

Actually, in the `custom_resource_tables` parameter, we provide the custom resource definition names. We expect the custom resources as query output at the query time. But since we have decided the parameter as `custom_resource_tables`, let's use custom resources in our language.

Added some examples showing how to match the full and short names.